sites/README.md

```diff
@@ -0,0 +1,293 @@
+# Valid properties
+
+List of possible values that can be used
+
+## name
+**(Mandatory)**
+
+**Type:** String
+
+This property will be used mainly for all the Kubernetes metadata name, it will also be used to set some properties as a default value if they have not been defined as tlsSecretName.
+
+Example:
+```
+name: canonical-com
+```
+
+## domain
+**(Mandatory)**
+
+**Type:** String
+
+This property define the project domain, it will be used for the Service, Deployment and Ingress objects.
+
+Example:
+```
+domain: canonical.com
+```
+
+## noIngress
+
+**Type:** Boolean
+
+**Default:** False
+
+When this property is set to true, the Ingress object will be empty.
+
+Example:
+```
+noIngress: true
+```
+
+## container
+**(Mandatory)**
+
+This property is for container related configurations. Here you can set the [image](#image) and the [tag](#tag).
+
+### image
+**(Mandatory)**
+
+**Type:** String
+
+Set the Docker image you want to use.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+```
+
+### tag
+
+**Type:** String
+
+**Default:** latest
+
+You can specify the tag you want to use with the image.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+    tag: a264efb326485
+```
+
+## wwwPrefixHost
```

Comment:
Rather than making this a choice, could we do logic to say that if it's got only 1 dot in it then we include www?

Basically, all apex domains (ubuntu.com, cloud-init.io) should have www, and all non-apex ones (design.ubuntu.com) shouldn't. We can enforce this pattern better through logic.

Reply:
Removed